docs/markdown/Rust-module.md

```diff
@@ -0,0 +1,32 @@
+---
+short-description: Rust language integration module
+authors:
+    - name: Dylan Baker
+      email: dylan@pnwbakers.com
+      years: [2020]
+...
+
+# Rust module
```

Comment:
This should mention that it is `unstable-rust` rather than `rust`, much like `unstable-simd`.